polling_stations/apps/data_finder/views.py

```diff
@@ -59,103 +60,114 @@ def form_valid(self, form):
 
 
 class BasePollingStationView(TemplateView, LogLookUpMixin):
+    __metaclass__ = ABCMeta
+
     template_name = "postcode_view.html"
 
+    @abstractmethod
+    def get_location(self):
+        pass
 
-class PostcodeView(BasePollingStationView):
+    @abstractmethod
+    def get_council(self):
+        pass
```

Comment:
Should these raise a [`NotImplementedError`](https://docs.python.org/2/library/exceptions.html#exceptions.NotImplementedError) rather then passing?